### ISSUE TITLE
Add Release files to dev branches

### ISSUE THREAD

**Issue by WSLUser** [WhitewaterFoundry/Pengwin#531]
**Describe the bug**
Unable to update to dev builds

**To Reproduce**
1. Install Pengwin from Store
2. Run pengwin-setup, it will run updates to pengwin-base stable. 
3. Run `sudo apt update` and `sudo apt upgrade -y`
4. Add the dev scripts for pengwin-base and pengwin-setup from [here](https://packagecloud.io/whitewaterfoundry)
5.  Run `sudo apt update again`.
6: See error:
```
E: The repository 'https://packagecloud.io/whitewaterfoundry/pengwin-base-dev/debian bullseye Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'https://packagecloud.io/whitewaterfoundry/pengwin-setup-dev/debian bullseye Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```
**Expected behavior**
I should be able to update to the dev branches.
**Screenshots**

**Additional context**
I won't be surprised if this is due to the shifting to Azure Pipelines but the fix is unrelated.

**Basic Troubleshooting Checklist**

[x] I have searched Google for the error message.
[x] I have checked official WSL troubleshooting documentation: https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#confirm-wsl-is-enabled.
[x] I have searched the official Microsoft WSL issues page: https://github.com/Microsoft/WSL/issues.
[x] I have searched the Pengwin issues page: https://github.com/WhitewaterFoundry/Pengwin/issues.
[x] I have reset Pengwin: Settings->Apps->Apps & features->Pengwin->Advanced Options->Reset. 
[ ] I have disabled and re-enabled WSL in Windows Features.
[ ] I have run Windows 10 updates and restarted.

What other troubleshooting have you attempted?

Insert here:

**Pengwin Version**

Find: Settings->Apps->Apps & features->Pengwin->Advanced Options->Version.

Insert here: 1.28
**Windows Build**

Run 'systeminfo | findstr /C:"OS"' in Command Prompt and insert here: OS Name:                   Microsoft Windows 10 Pro
OS Version:                10.0.18362 N/A Build 18362
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
BIOS Version:              American Megatrends Inc. E1781IMS.104, 7/31/2014

For help on retrieving: https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#check-your-build-number

**Comment by sirredbeard** [comment]
We haven't moved pengwin-setup/-base to Azure Pipelines, it is still building on CircleCI and pushing to PackageCloud.

That being said, I am not thrilled with how pengwin-setup/-base-dev perform when testing, apt issues happen and updates to the -dev packages can uninstall the -dev packages

We also need to establish three branches for pentwin-setup/-base and I want to build and run CI on all three.

The idea of having to do that on CircleCI is in part what prompted the switch to Azure Pipelines after I saw the potential in some conversations at Build.

So we will be moving pengwin-setup/-base to Azure Pipelines, after I finish the work on Fedora and Scientific Linux, and with that move I plan to make -dev work better.

The goal would be users who switch to -dev be able to stay on -dev with no bugs for being on -dev other than normal bugs that might be in -dev.

**Comment by WSLUser** [comment]
@crramirez Anything new with this? This still shows up and breaks `apt update` when running it outside of pengwin-setup.

**Comment by WSLUser** [comment]
I did notice we still have buster listed in `whitewaterfoundry.list`

**Comment by crramirez** [comment]
Yes, this is pending. And now I understand your issue. Please use `switch2dev` to change to the dev packages. This is a new script that pengwin-base ships with. Basically force the script to use buster.

**Comment by WSLUser** [comment]
Yep that worked. Hope we can get it somewhat on Bullseye soon.
